pkg/drivers/kic/oci/network_create_test.go

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright 2019 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package oci
+
+import (
+	"bytes"
+	"net"
+	"testing"
+)
+
+var dockerResponse string
+var dockerInspectGetterMock = func(name string) (*RunResult, error) {
+	var responseInBytes bytes.Buffer
+	responseInBytes.WriteString(dockerResponse)
+	response := &RunResult{Stdout: responseInBytes}
+
+	return response, nil
+}
+
+func TestDockerInspectWithMTU(t *testing.T) {
```

Comment:
how about putting this in a for loop that goes over the test cases ? feel free to say no if u think this is better, but here is an example
https://github.com/medyagh/minikube/blob/199ea3425b8343e533d8d1982ad750a8dab7d59e/cmd/minikube/cmd/status_test.go#L68

so maybe we could put both of these tests in one test and have a for loop of test cases?